phantomgames/Search/SearchGameViewModel.swift

```diff
@@ -7,7 +7,7 @@
 
 import UIKit
 
-protocol ViewModelDelegate: AnyObject {
```

Comment:
So you would actually want to keep this protocol with  generic name  of something like ViewModelDelegate or RefreshListdelegate or ShowErrorDelegate if you break them up because they are going to conform to ANY View Controller.
If you name it SearchGameViewModelDelegate, it implies it is specifically for this view controller..